apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -40,6 +40,8 @@ var _ webhook.Defaulter = &OpenTelemetryCollector{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (r *OpenTelemetryCollector) Default() {
+	opentelemetrycollectorlog.Info("default", "name", r.Name)
```

Comment:
Is this a leftover from a local test?

Reply:
nope just moving the logging from the end of the function

Reply:
Would it make sense to change it somehow? Perhaps even remove it?

Reply:
Alright -- the level is set to "info", but it's not providing any real info. It sounds more like a debug from your description. I'll let it at your discretion though.